conf/application.conf

```diff
@@ -88,6 +88,13 @@ microservice {
         host     = localhost
         port     = 9514
       }
+
+      contact-frontend {
+        protocol = http
+        host     = localhost
+        port     = 9250
+        serviceId = "plastic-packaging-tax-returns-frontend"
```

Comment:
whats this?

Reply:
Scaffold generated - I have removed as its not used.